client/blocks/time/display.js

```diff
@@ -32,7 +32,7 @@ const ornaments = [
 	'h', 'f', 'a', 't', 'n', 'c', 'o', 'p', 'e', 'r', 'k', 'l'
 ];
 
-const OrnamentedMonth = ({date}) => <Ornamented ornament={ornaments[date.monthIndex]} large>
+const OrnamentedMonth = ({date}) => <Ornamented ornament={ornaments[Math.min(date.monthIndex, 11)]} large>
```

Comment:
so, the ornaments were kind of chosen to match the odreian month names. maybe at some point we want to do configurable display styles for the time block?

Reply:
<img width="338" alt="screen shot 2018-04-20 at 21 54 01" src="https://user-images.githubusercontent.com/631757/39073374-7656dddc-44e5-11e8-9679-9730ee7007c8.png">

Reply:
Yeah having them configurable would be nice. They'll still work the same way for calendars with 12-month years